script/start_test_db.rb

```diff
@@ -2,7 +2,7 @@
 # frozen_string_literal: true
 
 BIN = "/usr/lib/postgresql/#{ENV["PG_MAJOR"]}/bin"
-DATA = "/tmp/test_data/pg"
+DATA = "tmp/test_data/pg"
```

Comment:
This was incorrectly creating the PG data folder in the `/tmp` directory instead of the `/var/www/discourse/tmp` directory.